Unix/scripts/installntlm-ubuntu16

```diff
@@ -0,0 +1,172 @@
+#!/bin/bash
+
+OMI_DEB=$(readlink -f ~/omi-1.1.0-2.rhel.x64.deb)
```

Comment:
There is a lot of Ubuntu specific code here. Maybe the next step will be to get it cross plat.